modules/openxr/openxr_api.cpp

```diff
@@ -590,41 +592,40 @@ bool OpenXRAPI::create_instance() {
 			}
 
 			// And record that we want to enable it (dependent extensions may be requested multiple times).
-			CharString ext_name = requested_extension.name.ascii();
```

Comment:
Is there a reason for moving this higher? It doesn't seem to be used until below this line.